File: StudentCredit.Application/Applications/Common/Dtos/CreditSelectFilterDto.cs
using StudentCredit.Data.Applications.Common.Enums;

namespace StudentCredit.Application.Applications.Common.Dtos
{
    public class CreditSelectFilterDto
    {
        public string Uin { get; set; }

        public CreditType CreditType { get; set; }

        public string CreditNumber { get; set; }

        public int BankId { get; set; }

        public int ApplicationOneId { get; set; }
    }
}

File: WebApplication/src/modules/application/common/models/credit-select-filter.dto.ts
import { CreditType } from "../enums/credit-type.enum";

export class CreditSelectFilterDto {
  uin: string;
  creditType: CreditType;
  creditNumber: string;
  bankId: number;
  applicationOneId: number;

  constructor(uin: string, creditType: CreditType, creditNumber: string, bankId: number, applicationOneId: number) {
    this.uin = uin;
    this.creditType = creditType;
    this.creditNumber = creditNumber;
    this.bankId = bankId;
    this.applicationOneId = applicationOneId;
  }
}